docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -22,6 +22,14 @@ On the consumer side, there are some common variations:
 
 The source of the events may be external to the system, such as physical devices in an IoT solution. In that case, the system must be able to ingest the data at the volume and throughput that is required by the data source.
 
+There are two primary approaches to structuring event payloads:
+
+- **Including all required attributes in the payload**: This approach is faster and more scalable, as consumers do not need to query the database. However, it can lead to data consistency issues due to multiple [systems of record](https://en.wikipedia.org/wiki/System_of_record), particularly after updates. Additionally, this method may result in [stamp coupling](https://en.wikipedia.org/wiki/Coupling_(computer_programming)) and bandwidth challenges at scale. Contract management and versioning can also become complex.
```

Comment:
```suggestion
- **Including all required attributes in the payload**: This approach is used when you desire consumers have all available information without the need query an external data source. However, it can lead to data consistency issues due to multiple [systems of record](https://wikipedia.org/wiki/System_of_record), particularly after updates. Contract management and versioning can also become complex.
```